devices.js

```diff
@@ -2792,6 +2792,28 @@ const devices = [
     },
 
     // Hive
+    {
+        zigbeeModel: ['MOT003'],
+        model: 'MOT003',
+        vendor: 'HiveHome.com',
+        description: 'Motion sensor',
+        supports: 'occupancy, temperature, battery, illuminance',
+        fromZigbee: [
+            fz.temperature, fz.ias_occupancy_alarm_1_with_timeout,
+            fz.illuminance, fz.hive_battery, fz.ignore_basic_report,
+        ],
+        toZigbee: [],
+        meta: {configureKey: 1},
+        configure: async (device, coordinatorEndpoint) => {
+            const endpoint = device.getEndpoint(6);
+            await bind(endpoint, coordinatorEndpoint, [
+                'msTemperatureMeasurement', 'genPowerCfg', 'msIlluminanceMeasurement',
+            ]);
+            await configureReporting.temperature(endpoint);
+            await configureReporting.batteryVoltage(endpoint);
```

Comment:
Can you change this line to `await configureReporting.batteryPercentageRemaining(endpoint);` and  `fz.hive_battery` to `fz.battery` and see if it works?

Reply:
Hey dude

I've made a few changes, at first I was only seeing the voltage, added some reading in (to populate the DB) and using the fz.battery works though :)

Reply:
Thanks, but I think 

```js
            const readPowerConfigClusers = [
                'batteryVoltage',
                'batteryPercentageRemaining',
                'batteryManufacturer',
                'batterySize',
                'batteryAHrRating',
                'batteryQuantity',
                'batteryRatedVoltage',
                'batteryVoltMinThres',
            ];
            await endpoint.read('genPowerCfg', readPowerConfigClusers);
```

and 

```js
await configureReporting.batteryVoltage(endpoint);
```

Are unneeded and can be removed. Battery percentage remaining is enough to know about the battery state.

Reply:
ah - I left these in just to populate the DB - and this does only during config, happy to remove most of them but would like to keep batteryPercentageRemaining as it would be good to have a way to trigger that to update manually (for instance if the sensor battery has been changed) - in all honesty I was looking to add config lines for a few other sensors to do this in another PR as I've changed a couple of batteries and it's either taken a while for this to update (>12 hours) or not updating at all

yeah, reporting the voltage probably isn't required

Reply:
Leaving only `batteryPercentageRemaining` is fine

Reply:
Whipped these out :)